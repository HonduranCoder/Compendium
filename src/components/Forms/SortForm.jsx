import React from 'react';

export default function SortForm({ sortPokemon, setSortPokemon }) {
  return (
    <div>
      <select
        aria-label="sort"
        value={sortPokemon}
        onChange={(e) => setSortPokemon(e.target.value)}
      >
        <option value="asc">ASC</option>
        <option value="desc">DESC</option>
      </select>
    </div>
  );
}
