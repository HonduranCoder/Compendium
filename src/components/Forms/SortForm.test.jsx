import { screen, render } from '@testing-library/react';
import SortForm from './SortForm.jsx';

it('test sort', () => {
  render(<SortForm />);
  const dropdown = screen.getByLabelText('sort');
  expect(dropdown).toMatchSnapshot();
});
