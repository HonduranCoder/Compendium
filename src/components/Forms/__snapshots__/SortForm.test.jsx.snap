// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test sort 1`] = `
<select
  aria-label="sort"
>
  <option
    value="asc"
  >
    ASC
  </option>
  <option
    value="desc"
  >
    DESC
  </option>
</select>
`;
